=== modified file 'Makefile.am'
--- old/Makefile.am	2011-07-15 09:26:50 +0000
+++ new/Makefile.am	2011-07-28 16:51:20 +0000
@@ -13,6 +13,7 @@
 libpower_la_CFLAGS = \
         $(UPOWER_CFLAGS) \
         $(INDICATOR_CFLAGS) \
+	-DHAVE_GSD \
         -Wall -Werror \
         -DG_LOG_DOMAIN=\"Indicator-Power\"
 libpower_la_LIBADD = \

=== modified file 'configure.ac'
--- old/configure.ac	2011-07-18 13:40:17 +0000
+++ new/configure.ac	2011-07-28 16:51:16 +0000
@@ -39,7 +39,7 @@
         gtk+-3.0 >= $GTK_REQUIRED_VERSION
         indicator3-0.4 >= $INDICATOR_REQUIRED_VERSION
 ])
-PKG_CHECK_MODULES([GSD],[gnome-settings-daemon>= $GSD_REQUIRED_VERSION],
+PKG_CHECK_MODULES([GSD],[gnome-settings-daemon >= $GSD_REQUIRED_VERSION],
                   [have_gsd=yes],[have_gsd=no])
 AM_CONDITIONAL([HAVE_GSD],[test x$have_gsd = xyes])
 

